prompb/types.proto

```diff
@@ -56,6 +56,8 @@ message ReadHints {
   string func = 2;    // String representation of surrounding function or aggregation.
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
+  repeated string group_by = 5; // List of label names of any fields used in a group-by aggregation
```

Comment:
grouping is what we use internally, so probably best to stick with that.

Reply:
sure